listenbrainz/webserver/static/js/src/player-pages/PlayerPage.tsx

```diff
@@ -115,6 +117,30 @@ export default class PlayerPage extends React.Component<
     );
   }
 
+  savePlaylist = async () => {
+    const { currentUser } = this.context;
+    if (!currentUser || !currentUser.auth_token) {
+      return;
+    }
+    const { newAlert, playlist } = this.props;
+    try {
+      const newPlaylistId = await this.APIService.createPlaylist(
+        currentUser.auth_token,
+        playlist
+      );
+      newAlert(
+        "success",
+        "Created playlist",
+        <>
+          Created a new public{" "}
+          <a href={`/playlist/${newPlaylistId}`}>instant playlist</a>
+        </>
+      );
+    } catch (error) {
+      newAlert("danger", "Error", error.message);
```

Comment:
Instead of just "Error", let's make this more user-friendly. How about "Could not save playlist" ?

Reply:
Done.